core/.env_sample

```diff
@@ -1,2 +1,7 @@
-MNEMONIC="your mnemonic here"
-SLACK_WEBHOOK="web hook for your slack app here"
\ No newline at end of file
+MNEMONIC=""
+SLACK_WEBHOOK=""
+TWILIO_SID=""
+TWILIO_AUTH=""
+TWILIO_FROM_NUMBER=""
+DRI_NUMBER1=""
+DRI_NUMBER2=""
```

Comment:
So all .env variables that start with `DRI_NUMBER` will get called? It's worth adding comments in the .env_sample to explain the env variables

Reply:
I know some of the vars are outside the scope but could you take a swing at it

Reply:
for sure but can you add a comment to a `.env`?

Reply:
Yes just precede it with a "#"